libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/MapboxCarNavigationManager.kt

```diff
@@ -95,4 +95,26 @@ class MapboxCarNavigationManager internal constructor(
         navigationManager.navigationEnded()
         navigationManager.clearNavigationManagerCallback()
     }
+
+    private fun onUpdateTrip(trip: Trip) {
+        if (mapboxNavigation?.getTripSessionState() == TripSessionState.STARTED) {
```

Comment:
Is this check necessary? I think it is not possible for route progress observer to be invoked when trip session is stopped.

Reply:
This was my original theory of the cause. The reason I still think it is possible is because the update comes from nav native and I did not find a guard inside MapboxTripSession

https://github.com/mapbox/mapbox-navigation-android/blob/e1ae50f4a5d3b357e29a3b0f65961d69a60b3c56/libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt#L337-L390